cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsCloudNioFileSystemProvider.scala

```diff
@@ -4,23 +4,17 @@ import java.nio.file.Path
 
 import cloud.nio.impl.drs.DrsCloudNioFileProvider.DrsReadInterpreter
 import cloud.nio.spi.{CloudNioFileProvider, CloudNioFileSystem, CloudNioFileSystemProvider, CloudNioPath}
-import com.google.auth.oauth2.OAuth2Credentials
 import com.typesafe.config.Config
-import net.ceedubs.ficus.Ficus._
-
-import scala.concurrent.duration.FiniteDuration
 
 class DrsCloudNioFileSystemProvider(rootConfig: Config,
-                                    authCredentials: OAuth2Credentials,
+                                    val drsCredentials: DrsCredentials,
```

Comment:
Had to make this a val so that the test code could confirm we're building with the correct TTL.